### ISSUE TITLE
Upgrade to W34-HC-IMJD-3.0.0 Current Conditions Not Present

### ISSUE THREAD

**Issue by gary-hammer** [steepleian/weewx-Weather34#3]
I upgraded to the current version and after configuring as I wanted, there is a blank Current Conditions.
I recalled editing archivedata.php.tmpl due to having no lightning data at my PWS.
That file is still the version that I edited, same date and same edits are present.

Also, rain is showing 0.00 while the last 24 hr shows 0.19, which is the amount that has fallen so far to day.
I'm attaching archivedata.php as .txt
[archivedata.txt](https://github.com/steepleian/weewx-Weather34/files/5439587/archivedata.txt)

Also attaching archivedata.php.tmpl as .txt
[archivedata.php.tmpl.txt](https://github.com/steepleian/weewx-Weather34/files/5439600/archivedata.php.tmpl.txt)

**Comment by balloonhotair** [comment]
Need to see the apache2 error log and the browser error log.

**Comment by balloonhotair** [comment]
Also did you do a full update including the weewx parts of weather34. The location of the archivedata.php file has moved to a new serverdata directory that resides in weather34 directory. Do you have this directory?

**Comment by gary-hammer** [comment]
I did an install as outlined in INSTALL_GUIDE.MD
I have /home/weewx/skins/Weather34/serverdata
It has only the archivedata.php.tmpl and airqualitydata.php.tmpl files in it.

There isn't a browser error log. I'm attaching the apache2 error log as gh-error.log

[gh-error.log](https://github.com/steepleian/weewx-Weather34/files/5441692/gh-error.log)

**Comment by balloonhotair** [comment]
The apache2 log has a lot of errors. Any chance I can access your website from the internet? Also please post your index.php and w34Combineddata.php. Also are the archivedata.php you posted above the one in serverdata? If not please post that one.

**Comment by balloonhotair** [comment]
The browser error log for firefox and chrome is really the console error log.

**Comment by gary-hammer** [comment]
> The apache2 log has a lot of errors. Any chance I can access your website from the internet? Also please post your index.php and w34Combineddata.php. Also are the archivedata.php you posted above the one in serverdata? If not please post that one.

No, it is not the one in serverdata, there isn't a archivedata.php in serverdata.

Yes, you can access the website.  weather.ghammer.net

**Comment by gary-hammer** [comment]
Ok, there is serverdata under both skins and in the web directory.
The web directory has archivedata.php attached as gh-archivedata.php.txt
[gh-archivedata.php.txt](https://github.com/steepleian/weewx-Weather34/files/5441776/gh-archivedata.php.txt)

w34Combineddata.php attached as gh-w34Combineddata.php.txt
[gh-w34CombinedData.php.txt](https://github.com/steepleian/weewx-Weather34/files/5441788/gh-w34CombinedData.php.txt)

**Comment by balloonhotair** [comment]
So right now Ian is dealing with some very serious family issues, and these errors are in his area usually. That said we should be able to resolve the errors I see in the console log.
Check your weather34 directory and if you have a file called updater.php.disabled rename it without the disabled part

**Comment by balloonhotair** [comment]
Sorry wrong file above I meant
weewxcron.php.disabled

**Comment by gary-hammer** [comment]
I shall check as soon as dinner is finished.

**Comment by gary-hammer** [comment]
Yes, in the html output directory there was a file and I renamed it to remove the disabled portion.
No, difference in the website.

I think perhaps if you can provide me with all locations, I will uninstall from weewx, make certain all directories are removed, reboot the server, then install again.

Something is just not right. It is getting partial data. I'd think it should be an all or nothing thing.

**Comment by balloonhotair** [comment]
Ok so that fixed one error. But it still can not find the file currentconditionsw34.php? Is there a file in weather34?

**Comment by balloonhotair** [comment]
So you should have the file since I can see it in the install and the browser is showing a 500 error on the currentconditionsw34.php which means there is a php error. Since I do not run that version of the file, I will have to get a copy of it and see were the php error is.

**Comment by balloonhotair** [comment]
So the version of currentconditionsw34.php is pretty messed up in the install. So I made changes so it loads on my system. Here is a copy that loads on my system. Just remove the .txt from the extension.  Try it and see what it does.
[currentconditionsw34.php.txt](https://github.com/steepleian/weewx-Weather34/files/5442373/currentconditionsw34.php.txt)

**Comment by gary-hammer** [comment]
Uploaded the file and Bob's your uncle.

Ok, so now it's only missing temp and rain.

**Comment by balloonhotair** [comment]
Ok, progress. Now I need to see the latest apache2 error log.

**Comment by balloonhotair** [comment]
Also is there a realtime.txt file in the web serverdata directory?

**Comment by balloonhotair** [comment]
If not check your weewx.conf file config entry in w34Realtime
 filename = /var/www/html/weewx/w34weather/serverdata/w34realtime.txt  (this is mind yours will be different but the important part you need to have serverdata directory in the path).

**Comment by gary-hammer** [comment]
Thanks for the rapid troubleshooting and fixes.

If I install this to a clean weewx install, are the corrections incorporated in the setup now?

**Comment by balloonhotair** [comment]
They should be now that the generation of the services.txt has been corrected.

**Comment by gary-hammer** [comment]
I'll give it a walk through Monday and let you know.
